application/templates/mail/request_from_accepted_to_open.plush.html

```diff
@@ -1,7 +1,5 @@
-<p>A requester on <a href="<%= uiURL %>"><%= appName %></a> no longer wants you to
-    fulfill that request.</p>
 <h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
-<p>The requester, <%= receiverNickname %>, knocked this request back to just being
-    open.  So, they are no longer expecting you to fulfill this request.</p>
-<br>
-<br>
+<p>
+    The requester, <%= receiverNickname %>, isn't ready after all to have you fulfill their request.
```

Comment:
Do we need to make this message sound more final?  I was thinking if it goes from accepted to open, the `requestor` is basically rejecting the offer.